helm-charts/splunk-otel-collector/templates/daemonset.yaml

```diff
@@ -94,6 +94,19 @@ spec:
             - name: fluentd-config-cri
               mountPath: /fluentd/etc/cri
         {{- else }}
+        - name: copy-old-checkpoint
+          image: {{ template "splunk-otel-collector.image.fluentd" . }}
```

Comment:
Let's use splunk-otel-collector image here so we don't pull an extra image if native logs collection enabled